pandas/io/excel/_base.py

```diff
@@ -811,8 +812,13 @@ class ExcelFile:
     _engines = {"xlrd": _XlrdReader, "openpyxl": _OpenpyxlReader, "odf": _ODFReader}
 
     def __init__(self, io, engine=None):
-        if engine is None:
+        if engine == "xlrd" or engine is None:
```

Comment:
Maybe just limit this to the `engine` is None case, since we aren't explicitly removing `xlrd`, so OK if a user explicitly says `engine="xlrd"` for now